examples/kafka-hub/integration_test_suite/main.bal

```diff
@@ -0,0 +1,132 @@
+// Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/file;
+import ballerina/io;
+import ballerina/lang.runtime;
+import ballerina/log;
+import ballerina/websub;
+import ballerina/websubhub;
+
+configurable string HUB = ?;
+configurable string TOPIC = ?;
+configurable string CALLBACK_URL = ?;
+configurable string RESULTS_FILE_PATH = ?;
+final string SECRET = "test123$";
+
+final websub:Listener websubListener = check new (9090);
+final readonly & json[] messages = [ADD_NEW_USER_MESSAGE, LOGIN_SUCCESS_MESSAGE];
+
+type STATUS "successful" | "failed" | "partial";
+final string[] & readonly documentationCsvHeaders = ["Label", "Sent Count", "Received Count", "Status"];
+
+public function main() returns error? {
+    _ = check initializeTests();
+    any[][] testResults = [];
```

Comment:
We can define a tuple type and use it instead of using `any`. 
Can replace in other places as well

Reply:
Fixed with [716155e2e0366d768f4a116e826a29b51c6cc58a](https://github.com/ballerina-platform/module-ballerina-websubhub/pull/695/commits/716155e2e0366d768f4a116e826a29b51c6cc58a)